apigw-lambda-dynamodb-sam-go/README.md

```diff
@@ -0,0 +1,74 @@
+# API Gateway REST API to AWS Lambda to Amazon DynamoDB
+
+This pattern explains how to deploy an AWS SAM application that includes an API Gateway, AWS Lambda Function, and writes items to an Amazon DynamoDB table. 
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+    
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+   ```bash
+   git clone https://github.com/aws-samples/serverless-patterns
+   ```
+2. Change directory to the pattern directory:
+   ```bash
+   cd apigw-lambda-dynamodb-go-sam
```

Comment:
the directory name here and the name of code directory should match. I'd suggest renaming the code directory to ```apigw-lambda-dynamodb-go-sam```.